Week1/ex1.js

```diff
@@ -63,13 +62,29 @@ const seedDatabase = async () => {
     try {
         await execQuery(`CREATE DATABASE IF NOT EXISTS meetup`);
         await execQuery(`USE meetup`);
-        await Promise.all([execQuery(CREATE_INVITEE_TABLE), execQuery(CREATE_ROOM_TABLE), execQuery(CREATE_MEETING_TABLE)]);
-
-        await Promise.all(invitees.map(inv => { execQuery(`INSERT INTO invitee SET ?`, inv) }));
-
-        await Promise.all(rooms.map(r => { execQuery(`INSERT INTO room SET ?`, r) }));
-
-        await Promise.all(meetings.map(m => { execQuery('INSERT INTO meeting SET ?', m) }));
+        execQuery(CREATE_INVITEE_TABLE);
+        execQuery(CREATE_ROOM_TABLE);
+        execQuery(CREATE_MEETING_TABLE);
+
+        await Promise.all(
+            invitees.map(inv => {
+                execQuery(`INSERT INTO invitee SET ?`, inv)
+            }
+            )
```

Comment:
Usually it's better to indent this way

```suggestion
            })
```